app/models/changelog_data.php

```diff
@@ -59,9 +59,97 @@
     '4.4'   => '2016-06-02',
     '5.0'   => '2016-08-04',
     '5.1'   => '2016-11-04',
+    '5.2'   => '2017-02-02',
 ];
 
 $changelog = [
+  '5.2' => [
+      [
+          'message' => ['Added link to Pootle or Pontoon under strings from the Aurora channel, in order to quickly edit translations'],
+          'section' => ['end_user'],
+          'type'    => ['new'],
+          'commit'  => ['6876a6d910f94b7f645ef3346aa3911ea2e37771'],
+          'authors' => ['SkySymbol'],
+      ],
+      [
+          'message' => ['Added L20N support by switching to compare-locales for parsing product repositories'],
```

Comment:
This is wrong, compare-locales doesn't support l20n (yet). I would say it's `code`, and note "Switched from Silme to compare-locales as library for parsing localization files in Mozilla products".